content/en/docs/components/pipelines/operator-guides/installation/_index.md

```diff
@@ -32,11 +32,35 @@ You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
 >
 > This workaround was verified on Minikube using `v2.0.0`, and aligns with community suggestions in [kubeflow/pipelines#9546](https://github.com/kubeflow/pipelines/issues/9546). It may also help users facing similar pod crash issues in other environments or newer versions.
 
+## Deploying Kubeflow Pipelines in Kubernetes Native API Mode
+
+Kubeflow Pipelines can be deployed in Kubernetes Native API mode, which stores pipeline definitions as Kubernetes Custom Resources instead of using external storage. This mode provides better integration with Kubernetes native tooling and GitOps workflows.
```

Comment:
```suggestion
Kubeflow Pipelines can be deployed in Kubernetes Native API mode, which stores pipeline definitions as Kubernetes Custom Resources instead of using external DB storage. This mode provides better integration with Kubernetes native tooling and GitOps workflows.
```